src/automation.md

```diff
@@ -108,12 +118,14 @@ jobs:
         TOKEN: ${{secrets.MY_SUPER_SECRET_STORED_KEY}}
 ```
 
-If this section stimulated your curiosity and you want to find out more, click [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-javascript-action).
+Still don't know how this .yml works? click [here](https://www.codeproject.com/Articles/1214409/Learn-YAML-in-five-minutes)
+
+If this section stimulated your curiosity and you want to find out more, click [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-a-javascript-action)
 
-Still don't know how this .yml works? click [here](https://www.codeproject.com/Articles/1214409/Learn-YAML-in-five-minutes).
+### Questions
 
 <authors-component v-bind:authors="[
     {
-      username: 'tiagoaguiar31',
-      name: 'Tiago Aguiar'
+      username: 'SheepFromHeaven',
```

Comment:
@SheepFromHeaven ok? Otherwise add correct responsible person (Tiago is not at VI anymore, right?)